GUI/gui.py

```diff
@@ -0,0 +1,197 @@
+    get_update_lines(subprocess.check_output("sudo torctl ip", shell=True))
+    get_update_lines(subprocess.check_output("sudo torctl status", shell=True))
+
+def start():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nSTART\n")
+    get_update_lines(subprocess.check_output("sudo torctl start", shell=True))
+    info_panel.configure(state="disabled")
+
+def stop():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nSTOP\n")
+    get_update_lines(subprocess.check_output("sudo torctl stop", shell=True))
+    info_panel.configure(state="disabled")
+
+def status():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nSTATUS\n")
+    get_update_lines(subprocess.check_output("sudo torctl status", shell=True))
+    info_panel.configure(state="disabled")
+
+def restart():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nRESTART\n")
+    get_update_lines(subprocess.check_output("sudo torctl restart", shell=True))
+    info_panel.configure(state="disabled")
+
+def autowipe():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nAUTO WIPE\n")
+    get_update_lines(subprocess.check_output("sudo torctl autowipe", shell=True))
+    info_panel.configure(state="disabled")
+
+def autostart():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nAUTO START\n")
+    get_update_lines(subprocess.check_output("sudo torctl autostart", shell=True))
+    info_panel.configure(state="disabled")
+
+def ip():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nWHAT IS MY IP?\n")
+    get_update_lines(subprocess.check_output("sudo torctl ip", shell=True))
+    info_panel.configure(state="disabled")
+
+def chngid():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nCHANGE TOR IDENTITY\n")
+    get_update_lines(subprocess.check_output("sudo torctl chngid", shell=True))
+    info_panel.configure(state="disabled")
+
+def chngmac():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nCHANGE MAC\n")
+    get_update_lines(subprocess.check_output("sudo torctl chngmac", shell=True))
+    info_panel.configure(state="disabled")
+
+def rvmac():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nREVERD MAC\n")
+    get_update_lines(subprocess.check_output("sudo torctl rvmac", shell=True))
+    info_panel.configure(state="disabled")
+
+def version():
+    info_panel.configure(state="normal")
+    info_panel.insert(tk.END, "\n\nVERSION\n")
+    get_update_lines(subprocess.check_output("sudo torctl version", shell=True))
+    info_panel.configure(state="disabled")
+
+def man():
+    messagebox.showinfo("Popup", MAN)
+
+def about():
+    messagebox.showinfo("Popup", ABOUT)
+
+def contact():
+    messagebox.showinfo("Popup", CONTACT)
+
+# Create main window
+window = tk.Tk()
+window.title("TORCTL by Aleff")
+
```

Comment:
Please don't take ownership of the project

Reply:
Replaced with `torctl - Redirect yout traffic`